hirte.spec.in

```diff
@@ -52,6 +52,7 @@ This package contains the node agent.
 
 %files
 %config %{_sysconfdir}/hirte/hirte.conf
+%config /usr/share/hirte/config/hirte-default.conf
```

Comment:
This is not configuration file in RPM language. Using %config would mean that if we post updated `/usr/share/hirte/config/hirte-default.conf` in updated RPM package, during upgrade existing `/usr/share/hirte/config/hirte-default.conf` would not be overriden and `/usr/share/hirte/config/hirte-default.conf.rpmnew` with the updated config would be created instead.

You need to use:

```
%{_datadir}/hirte/config/hirte-default.conf
```

Reply:
Changed